build.cake

```diff
@@ -91,10 +104,12 @@ Task("TestUsingExampleImplementation")
         DotNetCoreBuild(allSln, dotNetCoreBuildSettings);
 
         var allProjects = GetFiles(buildDir + "/*/*.csproj");
+        var parallelOptions = new ParallelOptions 
+        {
+            MaxDegreeOfParallelism = 16
```

Comment:
Why set this explicitly? I assume the TPL will automatically set this to a sensible option depending on your machine's configuration.